inference-engine/src/snippets/src/pass/collapse_subgraph.cpp

```diff
@@ -189,17 +184,29 @@ auto has_supported_in_out(std::shared_ptr<Node> n) -> bool {
         if (in.get_tensor().get_element_type() != ngraph::element::f32) {
             return false;
         }
+
+        if (in.get_partial_shape().is_static() && in.get_shape().size() > 6) {
```

Comment:
What if shapes are `dynamic`? Do we support such case?

Reply:
SnippetS are not tested on dynamic shapes right now so I assume no support at all

Reply:
done